packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js

```diff
@@ -315,7 +315,7 @@ function useMediaUploadQueue() {
           startUploading({ id });
 
           trackEvent('upload_media', {
-            file_size: newFile.size,
+            fileSize: newFile.size,
             file_type: newFile.type,
```

Comment:
No camel case here?

Reply:
`fileSize` was mistakenly changed to `fileSize` for `trackEvent` only at this place. Reverted to `file_siz` in https://github.com/google/web-stories-wp/pull/9683/commits/5178599c1879c14b4a6045d0c3da8b837c86a0d0 and verified that it did not accidentally happen anywhere else for `trackEvent`.